config/config.exs

```diff
@@ -119,6 +119,20 @@ config :skate, Skate.WarmUp,
   max_attempts: 20,
   seconds_between_attempts: 1
 
+config :skate, Oban,
+  repo: Skate.Repo,
+  peer: Oban.Peers.Postgres,
+  queues: [default: 10],
+  plugins: [
+    Oban.Plugins.Pruner,
+    {
+      Oban.Plugins.Cron,
+      crontab: [
+        {"*/5 * * * *", Skate.Oban.CleanUpNotifications, args: %{"cutoff_days" => 100}}
```

Comment:
**suggestion:** I think ultimately we do want the cutoff to be something closer to 10 days than 100 in practice given the issues we had had previously with adding the index, but I'm fine with starting at 100 and reducing it later.

Reply:
I dunno where I got 100 in my head, so I'll to change this back to 10.

Do you think we should wait to bump the limit up from 100 as well?

Reply:
I think it's probably fine either way. The only advantage to having it 100 initially would be putting a more restrictive bound on how much data the job will initially churn through, but if there's some issue we can always just revert the changes rather than wait.